rllib/agents/sac/sac_policy.py

```diff
@@ -111,10 +111,13 @@ def unsquash_actions(actions):
 
     squashed_stochastic_actions, log_pis = policy.model.get_policy_output(
         model_out, deterministic=False)
-    stochastic_actions = unsquash_actions(squashed_stochastic_actions)
+    stochastic_actions = squashed_stochastic_actions if config[
+        "normalize_actions"] else unsquash_actions(squashed_stochastic_actions)
```

Comment:
Which one should we use? Do either work?

Reply:
The normalize actions wrapper works better empirically.

Reply:
Ok, let's remove the squashing then and have sac.py enable normalize_actions by default.

Reply:
Just to make sure, even with the action normalization/rescaling, we still need to squash the actions in SAC. What we don't have to do is unsquash them on this level.

Ideally, I think SAC should be able to just handle different action ranges though and there's no reason why it couldn't do it. In the softlearning implementation, I just haven't bothered to change things because we pretty much automatically scale all the action spaces to be between `-1...1`.